src/components/graph/menus/create-node-menu.js

```diff
@@ -112,7 +112,7 @@ const CreateNodeMenu = ({
             action: () => buildNode(),
             id: 'buildNode',
             disabled:
-                activeNode?.data?.buildStatus === 'BUILT' ||
+                activeNode?.data?.buildStatus.startsWith('BUILT') ||
```

Comment:
Maybe ?.startsWith('BUILT') for safety ?

Reply:
Done